widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -367,6 +366,17 @@ public void emitState(CallState callState) {
                     } else {
                         hideUIOnCallEnd();
                     }
+
+                    if (callState.isAudioCall()) {
+                        chatButton.setEnabled(true);
```

Comment:
replace it all with `callState.isAudioCall() || callState.isVideoCall() || callState.is2WayVideoCall()`